src/utils/utils_log.c

```diff
@@ -0,0 +1,204 @@
+/*
+ *
+ * Copyright (C) 2023 Intel Corporation
+ *
+ * Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ */
+
+#define _GNU_SOURCE 1
+#include <stdarg.h>
+#include <string.h>
+#include <time.h>
+#ifdef _WIN32
+#include <windows.h>
+#else
+#include <sys/types.h>
+#include <unistd.h>
+#endif
+
+#include "utils_common.h"
+#include "utils_log.h"
+
+typedef struct {
+    int timestamp;
+    int pid;
+    LogLevel level;
+    LogLevel flushLevel;
+    FILE *output;
+} LoggerConfig;
```

Comment:
see how the struct definition looks like in UMF -rename to umf_logger_config_t etc

Reply:
dito

Reply:
I don't understand why we can't have one naming convention. It should be called "utils_log_config_t". I know we should clean this up in many existing places, but at least we shouldn't make it worse.